clients/tests/test_referral_credits.py

```diff
@@ -50,3 +51,42 @@ def test_referral_delete(self):
         client = Client.objects.get(first_name='Test Client')
 
         self.assertEqual(client.credit2, 0)
+
+    def _setup_test_data(self):
+        referredClient = Client.objects.create(
+            first_name="Referred Man",
+            referred_by=self.client
+            )
+        insurance = Insurance.objects.create(
+            main_claimant=self.client,
+            provider='Test Insurance',
+        )
+        Coverage.objects.create(
+            insurance=insurance,
+            claimant=self.client,
+            period=Coverage.CALENDAR_YEAR,
+        )
+        Claim.objects.create(
+            patient=referredClient,
+            insurance=insurance,
+            submitted_datetime=timezone.now())
+        Referral.objects.create(
+            client=self.client,
+            credit_value=1)
+
+    def test_cannot_receive_credit_for_duplicate_referral(self):
+        from clients.forms.forms import ReferralForm
```

Comment:
why is this import here?

Reply:
Originally it was so I could keep track of it since it's only used there, but it was overlooked in the commit